gamePage.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <script src="script.js" defer></script>
+    <title>Pro-To-Type Typing Speed Game</title>
+</head>
+
+<body>
+    <main>
+        <div class="linkToRules">
+            <a href="index.html">Back to rules</a>
+        </div>
+        <h2>Copy the text below: </h2>
+        <div id="textsToTypeContainer">
+            <div id="resultsPopup">
+                <h2>RESULTS:</h2>
```

Comment:
i think you should change this to just RESULTS because the : is making it look off center